app/src/main/java/com/edu/sun/oereminder/ui/timetracking/TimeTrackingPresenter.kt

```diff
@@ -8,4 +8,12 @@ import com.edu.sun.oereminder.ui.timetracking.TimeTrackingContract.View
 class TimeTrackingPresenter(private val timeSheetRepository: TimeSheetRepository) :
     BasePresenterImpl<View>(), Presenter {
 
+    override fun load() {
+
```

Comment:
xóa dòng trống đi, máy cái này viết 1 dòng cũng được, việc gì phải thêm  nhiều dòng thế.
`The first rule of functions is that they should be small. The second rule of functions is that they should be smaller than that
`